### ISSUE TITLE
Update Readme.md file to make it more presentable.

### ISSUE THREAD

**Issue by NiteeshK21** [NiteeshK21/HacktoberFest22#2]
Bring some changes to readme.md file giving a brief introduction of hacktoberfest2022 followed by a short introduction to the project.

**Comment by napalion** [comment]
@NiteeshK21, Could you assign me for the README? I would love to change it

**Comment by The-Debarghya** [comment]
Just one thing I wanna ask...do you want the brief about hacktoberfest to be above the project description? 
And by the way the mermaid code isn't rendering properly while viewing the readme.
Can you assign it to me?

**Comment by NiteeshK21** [comment]
> Just one thing I wanna ask...do you want the brief about hacktoberfest to be above the project description? And by the way the mermaid code isn't rendering properly while viewing the readme. Can you assign it to me?

Yes. Firstly start with brief about hacktoberfest followed by the project description. There isn't any need to assign issue to you. Make any changes as you wish.
